spynnaker8/extra_models/__init__.py

```diff
@@ -40,4 +44,7 @@
     'WeightDependenceAdditiveTriplet',
     'PfisterSpikeTriplet',
     'SpikeNearestPairRule',
-    'RecurrentRule', 'Vogels2011Rule']
+    'RecurrentRule', 'Vogels2011Rule',
+
+    # Variable rate Poisson
+    'SpikeSourcePoissonVariable']
```

Comment:
Personally I'd prefer to call this VariableSpikeSourcePoisson, but I understand why it might be preferable for it to be this way round.

Reply:
Understood, but then it should be VariablePoissonSpikeSource, but that doesn't fit in with PyNN!